flarestack/data/icecube/northern_tracks/nt_v005_p01.py

```diff
@@ -0,0 +1,65 @@
+from flarestack.data.icecube.ic_season import IceCubeDataset, get_dataset_dir
+from flarestack.data.icecube.northern_tracks import (
+    NTSeasonNewStyle,
+    get_diffuse_binning,
+)
+
+icecube_dataset_dir = get_dataset_dir()
+
+from flarestack.data.dataset_index import dataset_index
+
+nt_data_dir = icecube_dataset_dir + "northern_tracks/version-005-p01/"
```

Comment:
What type is icecube_dataset_dir? I think do an os.path.join or Pathlib equivalent would be more robust

Reply:
I had just copy pasted the NTv5.0 file but I agree. Gonna move to Pathlib and test it.